CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# Rubocop-Bitcrowd Changelog
```

Comment:
➕ on making it `.md`

How about also merging in the old non-md `CHANGELOG` file? Might be confusing to have both...

Reply:
![screen shot 2018-10-22 at 15 41 07](https://user-images.githubusercontent.com/206108/47295838-ee432a00-d610-11e8-8c1d-f568a75d7863.png)

the old file should be gone. maybe github didn't sync correctly?